bom/runtime/pom.xml

```diff
@@ -354,6 +354,11 @@
                 <artifactId>quarkus-elasticsearch-rest-client</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-elasticsearch-low-level-client</artifactId>
```

Comment:
Perhaps `-low-level-` is a bit too technical and might be dropped ? The documentation might say instead that `quarkus-elastic-search-client` provides a low-level REST client ElasticSearch support`, etc

Reply:
I agree, this is a temporal name.
The issue is that we already have an hidden `elasticsearch-rest-client` extension and that I think we must provides something better that only the low level client.
Namming will be reviewed before this PR move into the ready stage ;)

Reply:
Hi @loicmathieu thanks, I've just noticed it myself in my IDE :-) sorry for the noise